internalshared/configutil/listener.go

```diff
@@ -99,6 +99,11 @@ type Listener struct {
 	CorsAllowedOrigins    []string    `hcl:"cors_allowed_origins"`
 	CorsAllowedHeaders    []string    `hcl:"-"`
 	CorsAllowedHeadersRaw []string    `hcl:"cors_allowed_headers,alias:cors_allowed_headers"`
+
+	// Custom Http response headers
+	CustomResponseHeaders 	 map[string]map[string]string `hcl:"-"`
+	CustomResponseHeadersRaw interface{}                  `hcl:"custom_response_headers,alias:custom_response_headers"`
```

Comment:
I don't think you need an alias here.

Reply:
done! just out of curiosity, what is alias used for?